src/haproxy-sidecar/Dockerfile.template

```diff
@@ -0,0 +1,7 @@
+FROM docker
```

Comment:
Why do you need both an `haproxy` sidecar *and* an `haproxy-sidecar` sidecar?

Can their functionality be merged into one repo -- they seem strongly related.